preppal-fe/src/pages/recipe.tsx

```diff
@@ -0,0 +1,85 @@
+import { Container, Image, Stack, Row, Col, Card} from 'react-bootstrap';
+import {FaStar, FaRegStar, FaHeart} from 'react-icons/fa';
+const chickenfriedrice = require('../assets/chickenfriedrice.webp')
+
+export default function Recipe() {
+    return (
+        <Container className="d-flex ">
+        <html>
+            <h1>Chicken's Fried Rice</h1>
+            <Stack direction="horizontal">
+                <div className="p-2">
+                    {[...Array(4)].map(star => {
+                        return <FaStar/>
+                    })}
+                    <FaRegStar/> 4.2
+                </div>
+                <div className="p-2">
+                    43 Reviews
+                </div>
+                <div className="p-2">
+                    10 Photos
+                </div>
+            </Stack>
+            <div>No MSG required{"\n"}</div>
+            <Stack direction="horizontal">
+                <div className="p-2">
+                    Author: _
+                </div>
+                <div className="p-2">
+                    Date published: January 1, 2000
+                </div>
+            </Stack>
+            <Stack direction="horizontal" gap={4}>
+                <div>
+                    <FaStar/> Rate
+                </div>
+                <div><FaHeart/> Favourite</div>
+            </Stack>
+            
+            <div>
+                <Image src={chickenfriedrice} width={200}/>
```

Comment:
Instead of chicken fried rice, let's come up with a better default pic if users don't want to upload a picture